os_migrate/roles/export_subnets/tasks/main.yml

```diff
@@ -3,9 +3,12 @@
     auth: "{{ os_migrate_src_auth }}"
     auth_type: "{{ os_migrate_src_auth_type|default(omit) }}"
     region_name: "{{ os_migrate_src_region_name|default(omit) }}"
+    validate_certs: "{{ os_migrate_src_validate_certs|default(False) }}"
```

Comment:
We shouldn't default these options to False, that would be "insecure by default". We force this for running tests in `Makefile` via `-e`, but we shouldn't default this in code which is meant for production use.

Reply:
done